shrink_unshrink/models.py

```diff
@@ -223,6 +223,86 @@ def create_conv_dropout_model(conv1_filters=32,
   return model
 
 
+def create_conv_dropout_model_mfactor(conv1_filters=32,
+                              conv2_filters=64,
+                              dense_size=128,
+                              mfactor1=1.0,
+                              mfactor2=1.0,
+                              mfactor_dense=1.0,
+                              only_digits: bool = True) -> tf.keras.Model:
+  """Create a convolutional network with dropout.
+
+  When `only_digits=True`, the summary of returned model is
+  ```
+  Model: "sequential"
+  _________________________________________________________________
+  Layer (type)                 Output Shape              Param #
+  =================================================================
+  reshape (Reshape)            (None, 28, 28, 1)         0
+  _________________________________________________________________
+  conv2d (Conv2D)              (None, 26, 26, 32)        320
+  _________________________________________________________________
+  conv2d_1 (Conv2D)            (None, 24, 24, 64)        18496
+  _________________________________________________________________
+  max_pooling2d (MaxPooling2D) (None, 12, 12, 64)        0
+  _________________________________________________________________
+  dropout (Dropout)            (None, 12, 12, 64)        0
+  _________________________________________________________________
+  flatten (Flatten)            (None, 9216)              0
+  _________________________________________________________________
+  dense (Dense)                (None, 128)               1179776
+  _________________________________________________________________
+  dropout_1 (Dropout)          (None, 128)               0
+  _________________________________________________________________
+  dense_1 (Dense)              (None, 10)                1290
+  =================================================================
+  Total params: 1,199,882
+  Trainable params: 1,199,882
+  Non-trainable params: 0
+  ```
+  For `only_digits=False`, the last dense layer is slightly larger.
+
+  Args:
+    conv1_filters: The number of convolutional filters in the 1st convolutional
+      layer
+    conv2_filters: The number of convolutional filters in the 2nd convolutional
+      layer
+    dense_size: The number of neurons in the last dense layer
+    only_digits: If `True`, uses a final layer with 10 outputs, for use with the
```

Comment:
Let's add the new parameters to the arg section, along with descriptions, since this might be a little subtle

Reply:
The new commit does add some description

Reply:
thanks! Not used to reviewing code on GH, I thought the first commit was the lasts commit...